libs/dal/src/repositories/organization/community.organization.repository.ts

```diff
@@ -63,6 +65,10 @@ export class CommunityOrganizationRepository
   }
 
   async updateServiceLevel(organizationId: string, apiServiceLevel: ApiServiceLevelEnum) {
+    if (apiServiceLevel === ApiServiceLevelEnum.FREE) {
+      await this.integrationRepository.setRemoveNovuBranding(organizationId, false);
+    }
+
```

Comment:
This needs to be tied closely to `updateServiceLevel` since it needs to be always changed together